spyder/plugins/help/plugin.py

```diff
@@ -21,18 +21,23 @@
 from spyder.config.fonts import DEFAULT_SMALL_DELTA
 from spyder.plugins.help.confpage import HelpConfigPage
 from spyder.plugins.help.widgets import HelpWidget
+from spyder.plugins.mainmenu.api import ApplicationMenus, HelpMenuSections
 
 # Localization
 _ = get_translation('spyder')
 
 
+class HelpActions:
+    ShowSpyderTutorial = "spyder_tutorial_action"
+
+
 class Help(SpyderDockablePlugin):
     """
     Docstrings viewer widget.
     """
     NAME = 'help'
     REQUIRES = [Plugins.Console, Plugins.Editor]

```

Comment:
Since it requires one or the other or both maybe they could also be OPTIONAL, and check that at least onf them exists (on the check_compatibilty func?)